Sources/Bot/Models/RequiredStatusCheck.swift

```diff
@@ -0,0 +1,8 @@
+import Foundation
+
+public struct RequiredStatusChecks: Decodable, Equatable {
+    /// Require branches to be up to date before merging?
+    public let strict: Bool
```

Comment:
```suggestion
    public let isStrict: Bool
```

[Style-Guide](https://github.com/babylonhealth/ios-playbook/blob/master/Cookbook/Style-guide/README.md)

> boolean types should read like assertions (isLoading, isHidden, isFalse, etc)

Reply:
Done in 25a6004